plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -159,6 +165,80 @@ private ProblemDescriptor getStringConstructorProblem(PsiNewExpression expressio
       return null;
     }
 
+    private ProblemDescriptor getRedundantCaseEqualsProblem(PsiMethodCallExpression call) {
+
+      PsiExpression equalTo = call.getArgumentList().getExpressions()[0];
```

Comment:
Please wrap into `PsiUtil.skipParenthesizedExprDown()`.

Reply:
Thanks! Added PsiUtil.skipParenthesizedExprDown() and added a few tests for this case